.coderabbit.yaml

```
@@ -1,6 +1,19 @@
 reviews:
   profile: "assertive"
   request_changes_workflow: true
+  path_instructions:
+  - path: "lib/**/*.dart"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)